### ISSUE TITLE
SetOverlayInputMethod( m_ulOverlayHandle, vr::VROverlayInputMethod_Mouse) does nothing

### ISSUE THREAD

**Issue by DaveVanB** [ValveSoftware/openvr#1290]
I am working on a project that would place an Overlay into a running SteamVR application and provide "external" capabilities.  For most users, Keyboard and Mouse would be the primary input devices.
openvr seems to have numerous "Mouse" and "Keyboard" supporting functions, but none of them seem to work.
I'm sure I'm missing something simple, but even trying to run the sample applications, I am getting no positive results.
I have also read that openvr does not currently support mouse and keyboard as input devices, much less for an Overlay.  Can this really be true?
Thanks,
Dave

**Comment by elvissteinjr** [comment]
VROverlayInputMethod_Mouse refers to laser pointer interaction with simulated mouse functionality (left/right mouse buttons and scroll wheel).

SteamVR does not usually have input focus for keyboard and mouse to begin with. If you need that kind if input support, you'll have to grab these inputs yourself.

**Comment by DaveVanB** [comment]
Elvis, thanks for your answer and for confirming my suspicions.  I have found some add-ons that say they can turn your mouse into a "controller" but for the applications I'm trying to create, I prefer the precision and familiarity of a real mouse and cursor.
Are you aware of any example code that grabs the mouse?  I can live without the keyboard if I have to.

Also, for future readers of this thread, once I got my Dashboard overlays accepting input from the laser pointers, I spent hours trying to figure out how to get a non-Dashboard overlay to accept input.  Finally found the "Overlay.SetOverlayFlag(Handle, VROverlayFlags.MakeOverlaysInteractiveIfVisible, True)" setting of a flag seems to enable that.  
Thanks,
Dave
